src/main/java/org/assertj/vavr/api/VavrAssertions.java

```diff
@@ -13,10 +13,7 @@
 package org.assertj.vavr.api;
 
 import io.vavr.Lazy;
-import io.vavr.collection.List;
-import io.vavr.collection.Map;
-import io.vavr.collection.Multimap;
-import io.vavr.collection.Seq;
+import io.vavr.collection.*;
```

Comment:
Does it wrap imports if `length(package.name)>4`? Is there any ruleset defined?

Reply:
This may be due to my own settings. I can change it back if needed.